hoaxy/commands/site.py

```diff
@@ -205,6 +205,8 @@ class SiteCmd(HoaxyCommand):
              (--name=<n> | --domain=<d>) --alternate-domain=<d2>
   hoaxy site --disable (--name=<n> | --domain=<d>)
   hoaxy site --enable (--name=<n> | --domain=<d>)
+  hoaxy site --bulk-enable --exclusive (--names=<n>... | --domains=<d>...)
```

Comment:
`--exclusive` should be optional, not required. 
```bash
hoaxy site --bulk-enable [--exclusive] (--names=<n>... | --domains=<d>...)
```

Reply:
fixed in commit 4dcc4e1133e9760ea29c45de10398b54c6f755fb